### DISCUSSION TITLE
Adding a Victron Inverter Icon

### DISCUSSION THREAD

**Discussion by johanndurandsa** [slipx06/sunsynk-power-flow-card#183] [Ideas]
I Love the new Inverter Icons you adding to your card. I am helping out a friend converting him to your card for his Victron. Is there a possibility that you can add a Victron Inverter Icon also to your List?
![victron2](https://github.com/slipx06/sunsynk-power-flow-card/assets/98578816/7df53612-2e6d-497c-b2fa-2d67e11f197b)

**Comment by slipx06** [comment]
Hi. Yes I can add it. If there are any custom inverter status codes we need to add please share those as well.

**Reply by johanndurandsa** [reply]
Thanks, let me start playing with it and then will let you know

**Comment by slipx06** [comment]
Added to 4.7.0

**Comment by johanndurandsa** [comment]
Dankie Baie, Lyk goed

________________________________

**Comment by silviopen** [comment]
Hi, congratulations on the wide variety of options provided. I too would like to customize the inverter icon (mine is a Sungrow SH..).
Does it require a lot of work to give the user the option to upload an image, instead of choosing from the available ones?
Alternatively, could the image saved in a specific folder be replaced?

**Comment by slipx06** [comment]
Thanks. I have not really looked at that but the image needs to be correctly formatted and of specific dimensions which could be problematic for most users. Its relatively easy to add to the built in images. You can log a feature request if you would like this added.

**Comment by johanndurandsa** [comment]
Hi, just a question, why is the Solar Value showing like the Picture on a Victron System

[cid:8a7a2e59-195e-4b4b-9ed2-efe5176e9917]
________________________________

**Comment by slipx06** [comment]
I don’t see any picture?

**Comment by johanndurandsa** [comment]
Hi Sorry, Just saw your mail now,

Here is a Attachment.

Thanks again for your hard work on this card

Johann
________________________________

**Comment by drtinaz** [comment]
> Hi. Yes I can add it. If there are any custom inverter status codes we need to add please share those as well.

Here is status codes directly from Victron's modbus register list. Should cover all Vebus connected inverters.

0=Off
1=Low Power
2=Fault
3=Bulk
4=Absorption
5=Float
6=Storage
7=Equalize
8=Passthru
9=Inverting
10=Power assist
11=Power supply
244=Sustain
252=External control
